praw/models/reddit/live.py

```diff
@@ -431,3 +431,9 @@ def remove(self):
         url = API_PATH['live_remove_update'].format(id=self.update.thread.id)
         data = {'id': self.update.fullname}
         self.update.thread._reddit.post(url, data=data)
+
+    def strike(self):
+        """Strike a content of a live update."""
```

Comment:
Can we have an example please? If you feel like also adding an example for `remove` that would be great too (doesn't have to be in this pull request, but that's fine too).

Perhaps we could also indicate which attribute on an update indicates whether or not it has been stricken (that could probably be added elsewhere too -- I'll keep that in mind going forward).